models/games.models.js

```diff
@@ -5,3 +5,11 @@ exports.selectCategories = () => {
     return categories;
   });
 };
+
+exports.selectReviewById = (review_id) => {
```

Comment:
we tend to reject promises in here, which get picked up in the controller catch block, and passed to error handling middleware.